src/Generator/PdfOptionsGenerator.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\InvoicingPlugin\Generator;
+
+use Symfony\Component\Config\FileLocatorInterface;
+
+final class PdfOptionsGenerator implements PdfOptionsGeneratorInterface
```

Comment:
What do you think about this naming? 
```suggestion
final class PdfOptionsWithAllowedFilesProvider implements PdfOptionsProviderInterface
```